build.gradle

```
@@ -1,34 +1,35 @@
 plugins {
-	id 'org.springframework.boot' version '2.4.3'
-	id 'io.spring.dependency-management' version '1.0.11.RELEASE'
-	id 'java'
+    id 'org.springframework.boot' version '2.4.3'
+    id 'io.spring.dependency-management' version '1.0.11.RELEASE'
+    id 'java'
 }
 
 group = 'com.example'
 version = '0.0.1-SNAPSHOT'
 sourceCompatibility = '1.8'
 
 repositories {
-	mavenCentral()
+    mavenCentral()
 }
 
 dependencies {
-	// spring
-	implementation 'org.springframework.boot:spring-boot-starter-web'
-	implementation 'org.springframework.boot:spring-boot-starter-jdbc'
+    // spring
+    implementation 'org.springframework.boot:spring-boot-starter-web'
+    implementation 'org.springframework.boot:spring-boot-starter-jdbc'
+    implementation group: 'org.springframework.boot', name: 'spring-boot-starter-validation', version: '2.4.4'
```

```suggestion
    implementation 'org.springframework.boot:spring-boot-starter-validation'
```

validation 💯
버전을 명시하지 않아도 되는 이유는 뭘까요? (키워드: Spring Dependency Management)

---

mvnrepository 에서 그냥 복붙했어요..허허 부트에서 혼자 막 알아서 뚝딱뚝딱 해준다고 들었는데.. 내일 해당 키워드로 찾아보겠습니다!